content/en/blog/2024/collector-roadmap/index.md

```diff
@@ -65,7 +65,7 @@ wanted to focus on:
    OTLP exporter.
 2. Individual Go modules that the Collector components rely upon must also be
    marked as stable as per the project's
-   [versioning guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md#public-api-expectations).
+   [versioning guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md#general-go-api-considerations).
```

Comment:
@open-telemetry/go-approvers is this change ok?

Reply:
FWIW this is a @open-telemetry/collector-approvers question.

Reply:
This change is okay :) and thanks Tyler for redirecting